test/integration/organisation_test.rb

```diff
@@ -410,4 +417,29 @@ class OrganisationTest < ActionDispatch::IntegrationTest
     visit "/government/organisations/charity-commission"
     refute page.has_css?('.gem-c-heading', text: 'Our ministers')
   end
+
+  it 'shows the non-ministers for an organisation' do
+    visit "/government/organisations/attorney-generals-office"
+    assert page.has_css?(".gem-c-heading", text: "Our management")
+    assert page.has_css?('.gem-c-image-card .gem-c-image-card__title-link', text: 'Sir Jeremy Heywood')
```

Comment:
There's a couple of points in this file where it switches between single quotes and double quotes, should be more consistent.